src/components/EmailInviteDialog.js

```diff
@@ -174,24 +180,46 @@ const EmailInviteFriendsDialog = ({
       setPersonalMessageError(false)
     }
   }
+
+  let beforeShareImage
+  let afterShareImage
+  const alt = imgCategory
+  switch (imgCategory) {
+    case 'seas':
+      beforeShareImage = seasEmailInvite
```

Comment:
syntax but you don't need the double switch statement, you can just assign both values in 1 case 
```
case 'cats':
     beforeShareImage = seasEmailInvite
     afterShareImage = catsSent
     break
   ```